share/dictionary/der/dictionary

```diff
@@ -0,0 +1,21 @@
+# -*- text -*-
+# Copyright (C) 2022 The FreeRADIUS Server project and contributors
+# This work is licensed under CC-BY version 4.0 https://creativecommons.org/licenses/by/4.0
+# Version $Id$
+#
+#	The FreeRADIUS Vendor-Specific dictionary for TLS operations.
+#
+# Version:	$Id$
+#
+
+PROTOCOL	DER		11354911
+BEGIN-PROTOCOL DER
+
+$INCLUDE dictionary.test
```

Comment:
probably don't want test dictionaries in the default build.

the `src/tests/unit` stuff allows for local dictionaries to be loaded